src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1630,16 +1630,6 @@ public override BoundNode VisitUsingStatement(BoundUsingStatement node)
 
```

Comment:
I can't comment on the actual line, but why not call `base.VisitUsingStatement` unconditionally? And shouldn't the treatment for BoundLocalDeclarations do the same thing for all declaration kinds? Is there some reason to special case this in the `using` visitor?